app/views/pages/benevoles.html.erb

```diff
@@ -3,11 +3,40 @@
 <% end %>
 
 <div class="container my-5">
-  <h1>Bénévoles</h1>
+  <h1 class="mb-3">Bénévoles</h1>
 
-  <p>Covidliste est une initiative bénévole et citoyenne qui a vu le jour Il y a quelques semaines pour accélérer la campagne nationale de vaccination contre le Covid19 en distribuant les doses sur le point d'être jetées.</p>
+  <p>Covidliste est une initiative bénévole et citoyenne qui a vu le jour il y a quelques semaines pour accélérer la campagne nationale de vaccination contre le Covid19 en distribuant les doses sur le point d'être jetées.</p>
 
-  <p>Nous avons besoin de bénévoles pour venir renforcer nos rangs 💪</p>
+  <p>Merci à tous les bénévoles sans qui Covidliste ne pourrait pas exister !</p>
+
+  <p>Vous voulez nous rejoindre ? C'est en bas de la page !</p>
+
+  <div class="row my-5">
+    <% @volunteers.each do |volunteer| %>
+      <div class="col-6 col-lg-3 text-center mb-5">
+        <div>
+          <%= image_tag volunteer.picture_path, height: 150, loading: "lazy", class: "rounded-circle border", alt: volunteer.display_name %>
+        </div>
+        <div class="mt-3 <%= volunteer.anon ? "text-muted font-italic" : "" %>"><%= volunteer.display_name %></div>
+        <div class="small text-muted"><%= volunteer.bio %></div>
+        <div class="mt-1">
+          <% if !volunteer.twitter.blank? %>
+            <%= link_to icon("fab", "twitter"), "https://twitter.com/" + volunteer.twitter, class: "text-info", target: "_blank", rel: "noopener" %>
+          <% end %>
+          <% if !volunteer.linkedin.blank? %>
+            <%= link_to icon("fab", "linkedin"), "" + volunteer.linkedin, class: "text-info mx-2", target: "_blank", rel: "noopener" %>
+          <% end %>
+          <% if !volunteer.github.blank? %>
```

Comment:
Il existe la méthode `.present?` qui fait exactement l'inverse de `.blank?`, pour t'éviter de faire un `if not` 🙂 

```suggestion
          <% if volunteer.github.present? %>
```